CHANGELOG.md

```diff
@@ -1,3 +1,236 @@
+## v0.5.0 - 2020-10-23
+
+We're happy to announce the release of Lokomotive v0.5.0 (Eurostar).
+
+This release packs new features, bug fixes, code optimizations, platform updates, security hardening, and much more.
+
+### Changes in v0.5.0
+
+#### Kubernetes Updates
+
+- Update Kubernetes version to
+  [`v1.19.3`](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.19.md#v1193)
+  ([#1030](https://github.com/kinvolk/lokomotive/pull/1030)).
+
+#### Platform updates
+
+##### AKS
+
+- Update Kubernetes version to `1.18.8`
+  ([#1071](https://github.com/kinvolk/lokomotive/pull/1071)).
+
+##### Baremetal
+
+- Expose CNI MTU on the baremetal platform
+  ([#977](https://github.com/kinvolk/lokomotive/pull/977))
+
+#### New Components
+
+- Add component `headlamp`
+  ([#981](https://github.com/kinvolk/lokomotive/pull/981)).
+- Add component `inspektor-gadget`
+  ([#1076](https://github.com/kinvolk/lokomotive/pull/1076)).
+
+#### Component updates
+
+- Update Velero component for Packet [openebs and restic plugin support]
+  ([#881](https://github.com/kinvolk/lokomotive/pull/881)).
+- istio-operator: Update to 1.7.3
+  ([#1086](https://github.com/kinvolk/lokomotive/pull/1086)).
+
+#### Terraform Updates
+
+- Upgrade to Terraform 0.13
+  ([#824](https://github.com/kinvolk/lokomotive/pull/824)).
+
+#### Features
+
+- Add encryption to in-cluster pod traffic
+  ([#911](https://github.com/kinvolk/lokomotive/pull/911)).
+- aws,packet,baremetal: Remove rkt
+  ([#946](https://github.com/kinvolk/lokomotive/pull/946)).
+- labels taints format change
+  ([#1042](https://github.com/kinvolk/lokomotive/pull/1042)).
+- prometheus-operator: Add external_url
+  ([#964](https://github.com/kinvolk/lokomotive/pull/964)).
+
+#### Docs
+
+- Add document for admission webhook
+  ([#943](https://github.com/kinvolk/lokomotive/pull/943)).
+- docs/rook-ceph-storage: Use correct apply command
+  ([#1026](https://github.com/kinvolk/lokomotive/pull/1026)).
+- velero docs: Fix the formatting
+  ([#1045](https://github.com/kinvolk/lokomotive/pull/1045)).
+- docs/KEYS: switch to personal GPG key
+  ([#1047](https://github.com/kinvolk/lokomotive/pull/1047)).
+- Add a coding style guide
+  ([#953](https://github.com/kinvolk/lokomotive/pull/953)).
+- KEYS: Add knrt10 gpg key fingerprint
+  ([#1069](https://github.com/kinvolk/lokomotive/pull/1069)).
+- docs/development: Add link to coding style guide
+  ([#1059](https://github.com/kinvolk/lokomotive/pull/1059)).
+- docs/metallb: Clarify address_pools knob
+  ([#996](https://github.com/kinvolk/lokomotive/pull/996)).
+- Add how to guide on backing up and restoring rook-ceph volumes with Velero
+  ([#1048](https://github.com/kinvolk/lokomotive/pull/1048)).
+- docs: Fix headlamp attribute reference table
+  ([#1092](https://github.com/kinvolk/lokomotive/pull/1092)).
+- docs: openebs-operator link fix
+  ([#1101](https://github.com/kinvolk/lokomotive/pull/1101)).
+- docs/configuration-reference: remove "Example:"
```

Comment:
All the entries in the docs section should not have `docs` prefix before them. It is redundant.